docs/source/guides/monitoring.rst

```diff
@@ -3,13 +3,13 @@ Monitoring with Prometheus
 ==========================
 
 Monitoring stacks usually consist of a metrics collector, a time-series database to store metrics
-and a visualization layer. A popular stack is Prometheus_ with Grafana_ used as the visualization layer
+and a visualization layer. A popular stack of Prometheus_ with Grafana_ is used as the visualization layer
```

Comment:
this alters the meaning of this sentence. it should be `` A popular open-source stack is Prometheus used with Grafana as visualization tool to create rich dashboards``. Prometheus in itself is a system monitoring toolkit rather than a visualization tools.

Reply:
ok, i misunderstood the intention here.

Reply:
discussed with @aarnphm - we will merge and Aaron will address it in a separate PR

Reply:
cool